Packs/ServiceNow/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ServiceNow v2
```

Comment:
what about the pack version? need to bump it

Reply:
bumped